docs/strategy-callbacks.md

```diff
@@ -381,14 +382,14 @@ class AwesomeStrategy(IStrategy):
         dataframe, last_updated = self.dp.get_analyzed_dataframe(pair=pair,
                                                                 timeframe=self.timeframe)
         new_exitprice = dataframe['bollinger_10_upperband'].iat[-1]
-        
+
         return new_exitprice
 
 ```
 
 !!! Warning
-    Modifying entry and exit prices will only work for limit orders. Depending on the price chosen, this can result in a lot of unfilled orders. By default the maximum allowed distance between the current price and the custom price is 2%, this value can be changed in config with the `custom_price_max_distance_ratio` parameter.  
-    **Example**:  
+    Modifying entry and exit prices will only work for limit orders. Depending on the price chosen, this can result in a lot of unfilled orders. By default the maximum allowed distance between the current price and the custom price is 2%, this value can be changed in config with the `custom_price_max_distance_ratio` parameter.
+    **Example**:
```

Comment:
you'll need to be careful with trailing whitespace in markdown.

At least these 2 are "Meaningful" whitespaces - if there's 2 at the end, it'll be "new paragraph" - so now we're no longer rendering new lines, but all on one line ...

Reply:
Yep. I'll adjust linter settings and revert these unwanted changes. Thanks!